src/main/java/org/traccar/protocol/NavtelecomProtocolDecoder.java

```diff
@@ -305,6 +403,86 @@ protected Object decode(
                                 case 83:
                                     position.set("fuelTemp" + (j + 2 - 78), (int) buf.readByte());
                                     break;
+                                case 163:
+                                case 164:
+                                case 165:
+                                case 166:
+                                    value = buf.readShortLE();
+                                    if (value != (short) 0x8000) {
+                                        position.set(
+                                                Position.PREFIX_TEMP + "HighPrecision" + (j + 2 - 163),
```

Comment:
We don't use prefixes like this.

Reply:
The FLEX protocol has 8 temperature sensors with an accuracy of one degree aim. And 4 more pieces are high-precision, with an accuracy of 0.05 degrees. How should I name them to match the style?

Reply:
If you want to use the standard prefix, they should be `Position.PREFIX_TEMP + index`.